internal/apps/builder/builder.go

```diff
@@ -19,6 +19,9 @@ import (
 	"github.com/docker/docker/pkg/stdcopy"
 )
 
+// StaticSiteNginxImage is the nginx image used for static site container images.
+const StaticSiteNginxImage = "nginx:alpine"
```

Comment:
should we pin this to a specific alpine version (or at least a major version line)?